subschemas/substrate-api-proxy/src/resolvers/Query/council.ts

```diff
@@ -1,13 +1,17 @@
 import type { Context } from '../../types';
 import { BN, bnToBn } from '@polkadot/util';
 import type { BlockNumber } from '@polkadot/types/interfaces';
-import type {Council} from '../../generated/resolvers-types'
-
-interface CouncilInfo extends Omit<Council, 'members' | 'runnersUp' | 'candidates' | 'primeMember'> {
-  members: PartialCouncilMember[]
-  runnersUp: PartialCouncilMember[]
-  candidates: PartialCouncilCandidate[]
-  primeMember: PartialCouncilMember | null
+import type { Council } from '../../generated/resolvers-types';
+
+interface CouncilInfo
+  extends Omit<
```

Comment:
This formatting looks pretty bad 😆 

Maybe create a type just for `Omit<Council, 'members' | 'runnersUp' | 'candidates' | 'primeMember'>`

Reply:
Yeah, I will change the printwidth in prettier